hash/nes.xml

```diff
@@ -76709,7 +76709,7 @@ be better to redump them properly. -->
 
 <!-- Pirate fighting games -->
 
-	<software name="cityfgt4" supported="no">
+	<software name="cityfgt4">
```

Comment:
Should this be `supported="partial"` since sound emulation is incomplete, or doe it not use that hardware feature?